Corona-Warn-App/src/main/java/de/rki/coronawarnapp/ccl/dccwalletinfo/update/DccWalletInfoUpdateTask.kt

```diff
@@ -44,7 +44,8 @@ class DccWalletInfoUpdateTask @Inject constructor(
 
     data class Arguments(
         val dccWalletInfoUpdateTriggerType: DccWalletInfoUpdateTriggerType,
-        val startDelay: Long = 1_000L
+        val startDelay: Long = 1_000L,
+        val admissionScenarioId: String = ""
```

Comment:
I think in most cases, yes, but when the user selects a scenario it might lead to an error, and I think it is not nice to save it in this case. So I would pass it first and after a successful calculation it should be saved to subsequent calculations, does it make sense?